client/modules/PostEditor/PostEditor.tsx

```diff
@@ -119,6 +119,18 @@ function PostEditor(props: types.Props) {
     }
   );
 
+  const preSelectedCategory =
+    categories?.length > 0 ? _.head(categories) : null;
+  const selectedCategory =
+    categories?.length > 0
+      ? _.find(categories, (category) => {
```

Comment:
Almost there, try to use short-hand notation (from lodash documentation):
example: `_.find(users, { 'age': 1, 'active': true });`